### PR TITLE
Adding the ability to hide the tags header and display the tags list in block format

### PR DESCRIPTION
**Pull request by sebworks**
Adding the ability to hide the tags header and display the tags list in block format

## Changes

- Updates 'meta.less' file to include patterns for hiding the header and displaying the list in block format.
- Updates 'tags.html' file to allow users to pass in additional parameters which can be used to conditionally add classes.

## Review

- @jimmynotjim 
- @anselmbradford 
- @Scotchester 

et al

## Preview

inline
block

### CODE REVIEW

File: _includes/tags.html [hunk 1]
```diff
@@ -19,11 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}
 
+   display_header:     The optional boolean flag to display the tags_header element. 
+                       Defaults to True. Adds class 'tags__hide-header' if falsy 
+                       value passed (e.g., '', false, None)
```
Comment (anselmbradford) [inline _includes/tags.html]: If you passed false in in python is it `false` or `False` or does it not matter?

Comment (anselmbradford) [inline _includes/tags.html]: Missing final period for this comment, if that matters.

**Comment by anselmbradford** [comment]
:+1: LGTM 

One question, is https://github.com/cfpb/cfgov-refresh/pull/265/files#diff-66e3d5b7c115d8ac08688442088e9664R291 a TODO to double-check this value is correct?

**Comment by sebworks** [comment]
@anselmbradford , good catch. It should be False. Will add the period as well.

**Comment by jimmynotjim** [comment]
Merging this down and including it in the events list

**Pull request merged**